---a/net/minecraft/network/play/server/S19PacketEntityStatus.java
+++b/net/minecraft/network/play/server/S19PacketEntityStatus.java
@@ -1,48 +1,43 @@
 package net.minecraft.network.play.server;
 
 import java.io.IOException;
+
 import net.minecraft.entity.Entity;
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
-import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class S19PacketEntityStatus implements Packet
-{
+
+public class S19PacketEntityStatus implements Packet {
+
     private int field_149164_a;
     private byte field_149163_b;
     private static final String __OBFID = "CL_00001299";
 
-    public S19PacketEntityStatus() {}
+    public S19PacketEntityStatus() {
+    }
 
-    public S19PacketEntityStatus(Entity p_i46335_1_, byte p_i46335_2_)
-    {
+    public S19PacketEntityStatus(Entity p_i46335_1_, byte p_i46335_2_) {
         this.field_149164_a = p_i46335_1_.getEntityId();
         this.field_149163_b = p_i46335_2_;
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.field_149164_a = buf.readInt();
         this.field_149163_b = buf.readByte();
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeInt(this.field_149164_a);
         buf.writeByte(this.field_149163_b);
     }
 
-    public void func_180736_a(INetHandlerPlayClient p_180736_1_)
-    {
+    public void func_180736_a(INetHandlerPlayClient p_180736_1_) {
         p_180736_1_.handleEntityStatus(this);
     }
 
-    @SideOnly(Side.CLIENT)
-    public Entity func_149161_a(World worldIn)
-    {
-        return worldIn.getEntityByID(this.field_149164_a);
+    public void processPacket(INetHandler handler) {
+        this.func_180736_a((INetHandlerPlayClient) handler);
     }
-
-    @SideOnly(Side.CLIENT)
-    public byte func_149160_c()
-    {
-        return this.field_149163_b;
-    }
-
-    public void processPacket(INetHandler handler)
-    {
-        this.func_180736_a((INetHandlerPlayClient)handler);
-    }
-}
+}
